WBooks/MainTabBar/LibraryTab.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  LibraryTab.swift
+//  WBooks
+//
+//  Created by Gaston Maspero on 24/05/2019.
+//  Copyright © 2019 Wolox. All rights reserved.
+//
+
+import UIKit
+
+class LibraryTab: UITabBarController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureTabBar()
+        addNavBarButtons()
+    }
+    
+    private func addNavBarButtons() {
```

Comment:
You also need to set the status bar color to white

Reply:
The method `configureTabBar()` starts with `tabBar.barTintColor = .white`, which does that.

Reply:
I mean, the bar that is over to the navigationBar.
![image](https://user-images.githubusercontent.com/42350557/58436327-cfb77c80-8092-11e9-8f76-3b9ada26ecbd.png)